src/network/ports.rs

```diff
@@ -569,7 +570,16 @@ impl NewPort {
         set_name, with_name -> name: optional String
     }
 
-    // TODO(dtantsur): security groups
+    /// Add a new security group to the request.
+    pub fn add_security_group(&mut self, security_group_id: String) {
```

Comment:
Let us do it the same way cross-dependencies are handled. At some point we'll implement `SecurityGroup` as a first class object and will want to be able to use it here. So could update `types.rs` with a new transparent type and use `Into<SecurityGroupRef>` here? As a bonus we'll be able to use names here if `IntoVerified` is properly enabled.

It may sound confusing, just check other resources on how it's done.